### DISCUSSION TITLE
Where are the examples for https://arxiv.org/pdf/2110.14883.pdf？

### DISCUSSION THREAD

**Discussion by tongping** [hpcaitech/ColossalAI#2094] [Community | Q&A]
I am wondering where I can find the examples for Fig. Fig.17? It will be very beneficial if we could have the training script for Fig.17, better for both ColossalAI and DeepSpeed. I would like to see the performance results on our local machine.

**Comment by FrankLeeeee** [comment]
You can try this one https://github.com/hpcaitech/OPT-Benchmark.

**Comment by tongping** [comment]
Thanks a lot for the pointer.
